tor_ocr/main.py

```diff
@@ -36,6 +38,14 @@
 
     u_tor_post_id.reply(ocr_magic)
 """
+dotenv.load_dotenv()
+
+b_api = BlossomAPI(
+    email=os.environ.get('TOR_OCR_EMAIL'), 
+    password=os.environ.get('TOR_OCR_PASSWORD'), 
+    api_key='TOR_OCR_BLOSSOM_API_KEY', 
+    api_base_url=os.environ.get('TOR_OCR_BLOSSOM_API_BASE_URL')
+)
```

Comment:
There's a good chance we'll also need to specify the login API url at runtime as well, not just the base url.

Reply:
Yep, not sure how that got lost in the refactor, added it back in.